px-helm-mongo/4.deploy-mongo.md

```diff
@@ -0,0 +1,23 @@
+In this step, we will install Mongo using Helm.
+
+### Step: Install the Mongo chart
+
+Use Helm to install Mongo, with name _px-mongo_ and with the _persistence.storageClass_ parameter used to pass in the name of the PVC we just created.
+```
+helm install --name px-mongo \
+    --set persistence.existingClaim=px-mongo-pvc \
+    stable/mongodb
+```{{execute T1}}
+
+It'll take a few seconds and then Helm will report on the configurations it created. You will see that Helm creates a kubernetes secret, a service, a deployment, and a pod. To learn more about this Helm chart [check it out on github](https://github.com/kubernetes/charts/tree/master/stable/mongodb)!
+
+That's it! Isn't helm great? We think it is.
+
+### Step: Wait for Mongo to start
+
+Run the below command and wait for the Mongo pod to be in ready state.
+```
+watch kubectl get pods -o wide
+```{{execute T1}}
+
+When the Mongo pod is in STATE Running and READY 1/1 then hit ```clear```{{execute interrupt}} to ctrl-c and clear the screen. Be patient, if it's staying in Pending state for a while it's because it has to fetch the docker image.
```

Comment:
@BenHall  Is is possible to execute the interrupt when users hit Continue to move to the next step in the scenario? We want to avoid an explicit click on `clear` as above.